libs/dh/market-participant/feature-organization/src/lib/overview/dh-market-participant-organization-overview.component.html

```diff
@@ -82,8 +82,18 @@
         {{ t("columns.gridAreas") }}
       </mat-header-cell>
       <mat-cell *matCellDef="let element">
-        <!-- gridAreas is currently always undefined, but always returning the dash is correct for now. -->
-        {{ null | dhEmDashFallback }}
+        <ng-container *ngIf="element.actor !== undefined">
```

Comment:
Nitpick: This `ng-container` seems to be redundant as well. Consider just applying the `*ngIf` to the child instead. 
One of the cases where you wanna use `ng-container` is when an element/node already has a structural directive applied ex. `*ngFor` or `*ngIf` (because only a single structural directive, can be applied for an element/node) but this is not the case for `div.market-roles-cell`.